moulinette/actionsmap.py

```diff
@@ -511,8 +517,13 @@ def generate_cache(klass, namespaces=None):
             with open(am_file, 'r') as f:
                 actionsmaps[n] = yaml.load(f)
 
+            # clean old cached files
+            for i in os.listdir('%s/actionsmap/' % pkg.cachedir):
+                if i.endswith(".pkl"):
```

Comment:
no clear opinion.
even if I don't know glob prototype and behavior by heart, the code speaks for itself.
it has the advantage of becoming a one-liner.

no idea about performance difference, and no idea whether we really care about performance here (tend to think that we don't care), unless listdir clearly has flaws. but i would guess that glob is more or less listdir+a regex check behind the hood.